src/main/resources/META-INF/openapi.json

```diff
@@ -856,6 +865,15 @@
                     "checked": {
                         "description": "Whether or not this option was selected by the user",
                         "type": "boolean"
+                    },
+                    "risk": {
+                        "description": "Risk associated to this option",
+                        "enum": [
+                            "GREEN",
+                            "AMBER",
+                            "RED"
```

Comment:
Since `UNKNOWN` seems to be a valid value, could we add it to the openapi.json definition too, please?